draft-ietf-quic-transport.md

```diff
@@ -3299,8 +3532,7 @@ The two type bits from a Stream ID therefore identify streams as summarized in
 | 0x3      | Server-Initiated, Unidirectional |
 {: #stream-id-types title="Stream ID Types"}
 
-Stream ID 0 (0x0) is a client-initiated, bidirectional stream that is used for
-the cryptographic handshake.  Stream 0 MUST NOT be used for application data.
+The first bi-directional stream opened by the client is stream 0.
```

Comment:
For people who don't have the Stream 0 legacy (intended audience of this doc), is this statement even relevant?

Reply:
I'd double down and make this comprehensive.  We might have confusion about whether 0 is allowed if we don't say at least this much.